HW27/test_apidoc.py

```diff
@@ -0,0 +1,141 @@
+import pytest
+import jsonschema
+import requests
+
+
+@pytest.fixture
+def base_url():
+    return "https://restful-booker.herokuapp.com"
+
+
+@pytest.fixture
+def auth_token(base_url):
+    creds = {"username": "admin", "password": "password123"}
+    response = requests.post(f"{base_url}/auth", json=creds)
+    assert response.status_code == 200
+    return response.json()["token"]
+
+
+@pytest.fixture
+def created_booking(base_url):
+    booking_data = {
+        "firstname": "Nikita",
+        "lastname": "Test",
+        "totalprice": 150,
+        "depositpaid": True,
+        "bookingdates": {"checkin": "2025-05-10", "checkout": "2025-05-15"},
```

Comment:
datetime - will invalid and you will have to update it manually, better to generate it on a fly